invite/views.py

```diff
@@ -271,38 +271,43 @@ def invite(request):
         invite_item_formset = InviteItemFormSet(request.POST, request.FILES)
         if invite_item_formset.is_valid():
             for form in invite_item_formset.forms:
-                # make invite for each form
-                i = Invitation.objects.create(
-                    first_name=form.cleaned_data['first_name'],
-                    last_name=form.cleaned_data['last_name'],
-                    username=form.cleaned_data['username'],
-                    email=form.cleaned_data['email'],
-                    is_super_user=form.cleaned_data['is_super_user'],
-                )
-                i.custom_msg = (
-                    invite_item_formset
-                    .forms[0]
-                    .cleaned_data['greeting']
-                )
-                # set m2m relationships from initial object creation
-                permissions = (
-                    invite_item_formset
-                    .forms[0]
-                    .cleaned_data['permissions']
-                )
-                groups = (
-                    invite_item_formset
-                    .forms[0]
-                    .cleaned_data['groups']
-                )
+                try:
+                    # make invite for each form
+                    i = Invitation.objects.create(
+                        first_name=form.cleaned_data['first_name'],
+                        last_name=form.cleaned_data['last_name'],
+                        username=form.cleaned_data['username'],
+                        email=form.cleaned_data['email'],
+                        is_super_user=form.cleaned_data['is_super_user'],
+                    )
```

Comment:
What do you think about moving the end of the `try` to here, since this is where we would get the error if there was a hidden blank form?

Reply:
Yes, that makes sense. 👍🏼